config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "jira_workitem": "",
+        "coverall": "",
+        "coverll": "telefonicaid/fiware-facts",
+        "github": [
+            "telefonicaid/fiware-facts",
+            "telefonicaid/fiware-cloto"
+        ],
+        "docker": [
+            "fiware/bosun-cloto",
+            "fiware/bosun-facts"
+        ]
+    },
+    {
+        "enabler": "Business API Ecosystem",
+        "catalogue": "business-api-ecosystem-biz-ecosystem-ri",
+        "academy": "Business API Ecosystem",
+        "readthedocs": "business-api-ecosystem",
+        "helpdesk": "Business Ecosystem",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "FIWARE-TMForum/DSBILLINGMANAGEMENT",
+            "FIWARE-TMForum/Business-API-Ecosystem",
+            "FIWARE-TMForum/business-ecosystem-charging-backend",
+            "FIWARE-TMForum/DSCUSTOMER",
+            "FIWARE-TMForum/business-ecosystem-logic-proxy",
+            "FIWARE-TMForum/DSPRODUCTORDERING",
+            "FIWARE-TMForum/DSPARTYMANAGEMENT",
+            "FIWARE-TMForum/DSPRODUCTCATALOG2",
+            "FIWARE-TMForum/DSPRODUCTINVENTORY",
+            "FIWARE-TMForum/business-ecosystem-rss",
+            "FIWARE-TMForum/DSUSAGEMANAGEMENT"
+        ],
+        "docker": [
+            "fiware/business-api-ecosystem",
+            "conwetlab/biz-ecosystem-charging-backend",
+            "conwetlab/biz-ecosystem-logic-proxy",
+            "conwetlab/biz-ecosystem-rss"
+        ]
+    },
+    {
+        "enabler": "Cepheus",
+        "catalogue": "iot-data-edge-consolidation-ge-cepheus",
+        "academy": "",
+        "readthedocs": "fiware-cepheus",
+        "helpdesk": "Cepheus",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "coverll": "Orange-OpenSource/fiware-cepheus",
+        "github": [
+            "Orange-OpenSource/fiware-cepheus"
+        ],
+        "docker": [
+            "orangeopensource/fiware-cepheus-cep",
+            "orangeopensource/fiware-cepheus-broker"
+        ]
+    },
+    {
+        "enabler": "Cloud Portal",
+        "catalogue": "self-service-interfaces-cloud-portal",
+        "academy": "Cloud Portal",
+        "readthedocs": "fiware-cloud-portal",
+        "helpdesk": "Cloud Portal",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "ging/fiware-cloud-portal"
+        ],
+        "docker": [
+            "fiware/cloud-portal"
+        ]
+    },
+    {
+        "enabler": "Cloud Rendering",
+        "catalogue": "cloud-rendering",
+        "academy": "Cloud Rendering",
+        "readthedocs": "cloud-renderer",
+        "helpdesk": "Cloud Render",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "tospie/fiware-cloud-rendering-service"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Comet - Short Time Historic",
```

Comment:
"enabler": "Comet"

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585